documentation/sphinx/source/release-notes/release-notes-630.rst

```diff
@@ -10,9 +10,9 @@ Release Notes
 * Added ``commit_batching_window_size`` to the proxy roles section of status to record statistics about commit batching window size on each proxy. `(PR #4736) <https://github.com/apple/foundationdb/pull/4736>`_
 * The multi-version client now requires at most two client connections with version 6.2 or larger, regardless of how many external clients are configured. Clients older than 6.2 will continue to create an additional connection each. `(PR #4667) <https://github.com/apple/foundationdb/pull/4667>`_
 * Fix an accounting error that could potentially result in inaccuracies in priority busyness metrics. `(PR #4824) <https://github.com/apple/foundationdb/pull/4824>`_
-* Added ``cluster.bounce_impact`` section to status to report if there will be any extra effects when bouncing the cluster, and if so, the reason for those effects. `(PR #4770) <https://github.com/apple/foundationdb/pull/>`_
-* Added ``fetched_versions`` to the storage metrics section of status to report how fast a storage server is catching up in versions. `(PR #4770) <https://github.com/apple/foundationdb/pull/>`_
-* Added ``fetches_from_logs`` to the storage metrics section of status to report how frequently a storage server fetches updates from transaction logs. `(PR #4770) <https://github.com/apple/foundationdb/pull/>`_
+* Added ``cluster.bounce_impact`` section to status to report if there will be any extra effects when bouncing the cluster, and if so, the reason for those effects. `(PR #4847) <https://github.com/apple/foundationdb/pull/4847>`_
```

Comment:
Hmm, I hadn't really considered whether we should update the PR numbers when cherry-picking back, but I'll note that changing them on the release branch is not actually sufficient. This document is used to provide the release notes on the website from future branches as well, so if we made this change it would need to be on master and then cherry-picked back.

That seems like a bit of hassle, so maybe it's not worth changing them at all. I'm pretty sure there's precedent for not changing them, at least.

Reply:
Ok, I will revert the PR related change. Actually, I prefer not changing this PR-part when backporting (so we wouldn't need to update this part after creating a PR for the backport).

Reply:
Done.